front/src/routes/scene/edit-scene/triggers/SunriseSunsetTrigger.jsx

```diff
@@ -0,0 +1,78 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import { Text } from 'preact-i18n';
+
+import 'react-datepicker/dist/react-datepicker.css';
+import { RequestStatus } from '../../../../utils/consts';
+
+@connect('httpClient,user', {})
+class SunriseSunsetTrigger extends Component {
+  getHouses = async () => {
+    this.setState({
+      SceneGetHouses: RequestStatus.Getting
+    });
+    try {
+      const houses = await this.props.httpClient.get('/api/v1/house');
+      this.setState({
+        houses,
+        SceneGetHouses: RequestStatus.Success
+      });
+    } catch (e) {
+      this.setState({
+        SceneGetHouses: RequestStatus.Error
+      });
+    }
+  };
+
+  handleHouseChange = e => {
+    this.props.updateTriggerProperty(this.props.index, 'house', e.target.value);
+  };
+
+  componentDidMount() {
+    this.getHouses();
+  }
+
+  render({}, { houses }) {
+    let houseValid = false;
+    if (this.props.trigger.house === undefined || this.props.trigger.house === '') {
+      houseValid = true;
+    } else if (houses) {
+      const selectedHouse = houses.find(house => house.selector === this.props.trigger.house);
```

Comment:
I won't recommend doing this in the render, it'll slow down performance significantly.

Look at how we do this in other scene actions, we do it only when the house change/or when the list of houses change.